src/crypto/generichash/digest.rs

```diff
@@ -14,6 +15,13 @@ pub struct Digest {
     pub(super) data: [u8; crypto_generichash_BYTES_MAX as usize],
 }
 
+impl Digest {
+    /// Securely erase `Digest` using `utils::memzero`
+    pub fn drop_memzero(mut self) {
```

Comment:
I'd still this would be better served by making `memzero_value` a public API with a `Copy` bound.

I'd also say the doc comment wording is misleading as this is not "secure", it is merely best effort as `Digest` is as movable as any Rust type and could have been `memcpy`ied multiple times so that this will only zero out the last copy which will actually just be the `self` argument pass to this method without further compiler optimizations.

Reply:
Same does all the key wrappers and still they have a drop implementation with zeroing. Also, the decision to publish that function or not isn't up to me.

Reply:
> Same does all the key wrappers and still they have a drop implementation with zeroing. 

I am aware of that but they do not say anything about "securely" erasing things in their doc comments and I think we should avoid that here as well.

Reply:
Then please propose doc-comment that in your opinion wouldn't be confusing